ansible/pbTestScripts/testScript.sh

```diff
@@ -56,36 +77,56 @@ destroyVM()
 	vagrant destroy -f
 }
 
-# Takes in OS as arg 1
+# Takes in OS as arg 1, branchName as arg 2
 searchLogFiles()
 {
-	cd ~/adoptopenjdkPBTests/logFiles
-	if grep -q 'failed=[1-9]' *$1.log
+	cd $HOME/adoptopenjdkPBTests/logFiles
+	if grep -q 'failed=[1-9]' *$2.$1.log
 	then
 		printf "\n$1 Failed\n"
-	elif grep -q '\[ERROR\]' *$1.log
+	elif grep -q '\[ERROR\]' *$2.$1.log
 	then
 		printf "\n$1 playbook was stopped\n"
 	else
 		printf "\n$1 playbook succeeded\n"
 	fi
 }
 
+# Takes in the URL passed to the script
+splitURL()
+{
+	# breaks down url to array, and extracts info
+	IFS='/' read -r -a array <<< "$1"
+	if [ ${array[@]: -2:1} == 'tree' ]
+	then
+		branchName=${array[@]: -1:1}
+		folderName=${array[@]: -3:1}
+		unset 'array[${#array[@]}-1]'
+		unset 'array[${#array[@]}-1]'
+		for I in "${array[@]}"
+		do
+			gitURL="$gitURL$I/"
```

Comment:
I think you're overriding this var each time, is that the intent?

Reply:
Yep, it's to build back up the required git URL, without "tree" or the branch name in it.
i.e.
`https://github.com/AdoptOpenJDK/openjdk-infrastructure/tree/ccache` --> `https://github.com/AdoptOpenJDK/openjdk-infrastructure/`